spec/features/adventures/user_views_adventure_spec.rb

```diff
@@ -67,7 +92,33 @@
 
   scenario "authenticated user successfully switches an adventure from public" \
   " to private by clicking on the appropirate icon" do
+    user = FactoryGirl.create(:user)
+    visit new_user_session_path
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.